numba_dppy/examples/vectorize.py

```diff
@@ -55,5 +55,21 @@ def main():
     print("Done...")
 
 
+@vectorize([float64(float64, float64)], target="dppy")
```

Comment:
Is not the `target` keyword deprecated in Numba?

Reply:
I do not think it is deprecated for `@vectorize`. Numba master still has this test for Cuda: https://github.com/numba/numba/blob/master/numba/cuda/tests/cudapy/test_vectorize_decor.py#L27